src/components/layers-panel/layers-panel.tsx

```diff
@@ -193,6 +222,18 @@ export const LayersPanel = ({
     );
   };
 
+  const handleSelectMaps = (id: string): void => {
+    let newSelectedMapIds = selectedMapIds;
```

Comment:
For the base map you don't need to handle array but only one selected map style.